apps/analysis_framework/schema.py

```diff
@@ -311,6 +310,7 @@ class Query:
             page_size_query_param='pageSize'
         )
     )
+    analysis_framework_roles = graphene.List(graphene.NonNull(AnalysisFrameworkRolesListType))
```

Comment:
In general, we use `....ListType` for pagination. For this, this should be
```suggestion
    analysis_framework_roles = graphene.List(graphene.NonNull(AnalysisFrameworkRoleType), required=True)
```